crmsh/help.py

```diff
@@ -129,6 +125,48 @@ def __repr__(self):
         return str(self)
 
 
+class LazyHelpEntryFromCli(HelpEntry):
+    def __init__(
+            self,
+            short_help: str,
+            cmd_args: typing.Sequence[str],
+            alias_for=None, generated=False,
+    ):
+        super().__init__(short_help, '', alias_for, generated)
+        self._cmd_args = cmd_args
+
+    @functools.cached_property
+    def long(self):
+        args = ['crm']
+        args.extend(self._cmd_args)
+        args.append('--help-without-redirect')
+        rc, stdout = ShellUtils.get_stdout(args, shell=False)
+        return stdout
+
+    def paginate(self):
+        page_string('{}\n\n{}'.format(self.short, self.long))
+
+
+@dataclasses.dataclass
+class SubcommandTreeNode:
+    name: str
+    help: HelpEntry
+    children: dict[str, typing.Self]
+
+    def is_leaf(self) -> bool:
+        return bool(self.children)
```

Comment:
Typically, a leaf node is a node with, or without children?

Reply:
I removed this method as it is not used. In recursions, it is more straightforward to loop over `children` directly without checking whether it is a leaf node.